src/cljs/athens/devcards/app_toolbar.cljs

```diff
@@ -187,14 +198,17 @@
        (when @import-modal-open?
          [:div (use-style modal-style)
           [modal/modal
-           {:title [:div.modal__title [:> mui-icons/Publish] [:h4 "Import to Athens"] [button
-                                                                                       {:on-click-fn #(reset! import-modal-open? false)
-                                                                                        :label [:> mui-icons/Close]}]]
+           {:title [:div.modal__title [:> mui-icons/Publish]
+                    [:h4 "Import to Athens"]
+                    [button
+                     {:on-click-fn #(reset! import-modal-open? false)
+                      :label [:> mui-icons/Close]}]]
             :content [:div (use-style modal-contents-style)
                       ;; TODO: Write intro copy
                       [:p "Some helpful framing about what Athens does and what users should expect. Athens is not Roam."]
                       [features-table]
                       ;; TODO: Create browser file dialog and actually import stuff
-                      [:div [button-primary {:label "Add Files"}]]]
+                      [:div [:input {:type "file" :on-change file-cb}]]]
```

Comment:
We can just use [a hidden input field](https://github.com/athensresearch/athens/pull/286/files/8a49b2a50b26c739180c36630f20d0ef515fc7e9#r457205984) and a button for style consistencies